src/repositories/LoginRegisterRepoInterface.ts

```diff
@@ -0,0 +1,11 @@
+import { QueryResult } from 'pg';
+
+export default interface LoginRegisterRepoInterface{
```

Comment:
This is so well written and thought through, well done bro!!

Reply:
I knoe it is big but it's descriptive 🤣

Reply:
know*